data/examples/declaration/value/function/case-single-line-out.hs

```diff
@@ -3,5 +3,4 @@ foo x = case x of x -> x
 
 foo :: IO ()
 foo = case [1] of [_] -> "singleton"; _ -> "not singleton"
-
 foo = case [1] of { [] -> foo; _ -> bar } `finally` baz
```

Comment:
This change is caused by the fact that the second `foo` definition is grouped together with the type signature.